lib/spack/spack/cmd/create.py

```diff
@@ -333,15 +333,15 @@ class OctavePackageTemplate(PackageTemplate):
     # FIXME: Add additional dependencies if required.
     # depends_on('octave-foo', type=('build', 'run'))"""
 
-    def __init__(self, name, *args):
+    def __init__(self, name, *arg, **kwargss):
```

Comment:
I would use `args`, not `arg`. Same below

Reply:
Ahh.  How embarrassing.  Thanks!